airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -997,9 +1042,6 @@ def execute(self, context):
                 labels=self.labels,
                 encryption_configuration=self.encryption_configuration
             )
-        except HttpError as err:
```

Comment:
Is there no need to catch `Conflict` anymore? Are these errors ignored because of `exists_ok=True` argument of hooks' method `create_empty_table`?

Reply:
This operator as well as underlying hook method is deprecated and is using this one instead:
```
        self.create_empty_table(
            table_resource=table.to_api_repr(),
            project_id=project_id,
            location=location,
            exists_ok=True
        )
```
Idempotency is assured by `exists_ok=True`.